assets/components/aws-ebs-csi-driver/manifests/values.yaml

```diff
@@ -0,0 +1,89 @@
+# Default values for aws-ebs-csi-driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+image:
+  repository: amazon/aws-ebs-csi-driver
+  tag: "v0.5.0"
+  pullPolicy: IfNotPresent
+
+sidecars:
+  provisionerImage:
+    repository: quay.io/k8scsi/csi-provisioner
+    tag: "v1.5.0"
+  attacherImage:
+    repository: quay.io/k8scsi/csi-attacher
+    tag: "v1.2.0"
+  snapshotterImage:
+    repository: quay.io/k8scsi/csi-snapshotter
+    tag: "v2.1.1"
+  livenessProbeImage:
+    repository: quay.io/k8scsi/livenessprobe
+    tag: "v1.1.0"
+  resizerImage:
+    repository: quay.io/k8scsi/csi-resizer
+    tag: "v0.3.0"
+  nodeDriverRegistrarImage:
+    repository: quay.io/k8scsi/csi-node-driver-registrar
+    tag: "v1.1.0"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+# True if enable volume scheduling for dynamic volume provisioning
+enableVolumeScheduling: false
+
+# True if enable volume resizing
+enableVolumeResizing: false
+
+# True if enable volume snapshot
+enableVolumeSnapshot: false
+
+# False if you want to use a different storage option, e.g. OpenEBS
+enableDefaultStorageClass: true
```

Comment:
This should be false by default or we need to always render this property in values in `component.go`, as otherwise it will always be enabled.

Also, can we please move this at the bottom of the file, so we can keep upstream code separate from our changes?

Reply:
Fixed the rendering issue and moved :)